docs/specs/src/hwp_1_intro.md

```diff
@@ -0,0 +1,212 @@
+  rather than agreement proceeding in lockstep.)
+
+2.  **Coordination is grammatic**, in that it comes from embodying a
+  geometry that removes uncertainty, and embodying an ability to
+  compose different coordinative subsystems that have different
+  ground-rules.
+
+Axiom 1 arises from the insight that we cannot fight against the
+physical reality of different experiences by different nodes in the
+context of networked interaction nor fight against the real-world
+impossibility of determining global temporal order because of the
+non-existence of simultaneity. Instead, any coordination system must
+align its ontology with the truth that **global state actually does not
+exist**. Thus we start with what does actually exist: local temporal
+state. This local state can be shared with, and validated by, others in
+conformity with pre-defined ground-rules. In so doing we can still
+achieve difficult and complex coordination safely (including in the
+context of problems on the scale of global monetary transactions)
+without the costs and bottlenecks that arise from starting from the
+ontology of a single shared global state. Holochain is an implementation
+of a system using this alternate frame.
+
+Axiom 2 arises from the insight that systems for successful large scale
+coordination demand the property of anti-fragility, that is, they must
+perform better under perturbation[^2]. Coordination happens in the
+context of fundamentally dynamic environments in which the coordinating
+elements are changed by the fact of their coordination. Coordination is
+a co-evolutionary context. We claim by this axiom that what meets the
+challenge of anti-fragility in such contexts is composable sub-systems,
+in which the composition comes out of a grammatics that embodies the
+actual dimensionality of the problem subdomains (i.e. their geometry),
+and by which agents in that context can react powerfully to
+perturbations because the available composability is dimensionally
+aligned.
+
+The core axiom (though not explicitly stated as such) of the Byzantine
+Generals Problem is that coordination starts AFTER "consensus on state",
+i.e. that the Lieutenants can't execute their plan until they have
+followed the consensus algorithm and arrived at single data reality[^3].
+This axiom leads system designers to figure out how to implement
+machinery for **Global Consensus**. Our axioms lead us, instead, to
+implement tooling for **Scaling Consent** as an alternate solution to
+the Byzantine Generals Problem.
+
+From Global Consensus to Scaled Consent
+---------------------------------------
+
+In distributed systems, we believe that it is absolutely fundamental to
+understand that every action taken by an agent in any social context,
+happens because that agent has crossed a confidence threshold of some
+sort, from its own point of view, that the given action is appropriate
+to take. Stated another way: agentic assessment of the social context
```

Comment:
```suggestion
to take. Stated another way: agent-centric assessment of the social context
```

Is "agentic" synonymous with "agent-centric"? I think if you use it, it would help to introduce it as such. I see a few references of each throughout the documents.

Reply:
Ah, I didn't realize "agentic" is a real word:

> That behaves like an agent: able to express or expressing agency or control on one's own behalf or on the behalf of another

Reply:
Where did you find that, @maackle?
I just found
```
agentive | ˈājən(t)iv |
adjective
formal taking an active role; producing an effect: the program envisions designers as agentive, social citizens.
• Grammar denoting a word or part of a word that expresses the doer of an action: agentive nouns are typically derived from a verb through suffixation of -er.
```

Reply:
Well, on Wiktionary. But also I see it used in some psychological contexts.

Reply:
I like agentic more than agentive.  I think more people will resonate with it.  But I could be wrong.